elements/urth-viz-area/urth-viz-area.html

```diff
@@ -0,0 +1,121 @@
+<!--
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+-->
+<link rel="import" href="../urth-viz-chart/urth-viz-chart.html">
+
+<!--
+Represents a widget that draws a area chart.
+The chart accepts data via attribute `datarows` and column headers via attribute `columns`:
+
+1. `datarows` should be of type Array of Arrays, representing list of rows.
+
+2. 'columns' should also be of type Array of strings, representing column headers.
+
+@group Urth Viz
+@element urth-viz-area
+-->
+
+<dom-module id="urth-viz-area">
+
+    <template>
+        <urth-viz-chart id="chart" type="area" datarows="[[datarows]]" columns="[[columns]]" palette="[[palette]]" selection="{{selection}}" primary="{{primary}}" secondary="{{secondary}}">
+            <content></content>
+        </urth-viz-chart>
+    </template>
+
+    <script>
+
+    'use strict';
+
+    Polymer({
+        is: 'urth-viz-area',
+
+        properties: {
```

Comment:
I think that you can put this properties on a light common behavior to avoid repetition. I would put that behavior in the `urth-viz-chart` module.